src/runtime/virtcontainers/clh.go

```diff
@@ -736,7 +736,7 @@ func (clh *cloudHypervisor) check() (err error) {
 	defer func() { err = errorContext("clh.check", err) }()
 
 	cl := clh.client()
-	ctx, cancel := context.WithTimeout(context.Background(), clhAPITimeout*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Given your comments that the monitor thread would `check` every seconds every second, while we are giving each check a timeout of `10*1` second. Is it an issue?